### PR TITLE
Improve documentation of git_sitrep

### PR DESCRIPTION
**Pull request by llrs**
Improves documentation to explain what does it mean to be Vaccinated in this sitrep.

I haven't redocumented the man folder so I expect this to fail on travis

### CODE REVIEW

File: R/git.R [hunk 1]
```diff
@@ -449,7 +449,7 @@ use_git_credentials <- function(credentials) {
 #'
 #' Get a situation report on your current git/GitHub status. Useful for
 #' diagnosing problems.
-#'
+#' To vaccinate (have a general git ignore file) use [edit_git_ignore()]
```
Comment (jennybc) [inline R/git.R]: This should refer people to the `git_vaccinate()` function.

Comment (llrs) [reply R/git.R]: I didn't even knew that this function existed.

**Comment by jennybc** [comment]
Thanks!

**Comment by llrs** [comment]
Glad it helped!

**Pull request merged**